src/AbstractPluginManager.php

```diff
@@ -31,6 +33,15 @@
  *
  * The implementation extends `ServiceManager`, thus providing the same set
  * of capabilities as found in that implementation.
+ *
+ * @see ServiceLocatorInterface
```

Comment:
I use `@see` because the codesniffers out there as well as phpstorm are not able to detect the usage from within `@psalm-import-type`.